packages/cli/src/lib/implementation/global.utils.ts

```diff
@@ -40,7 +40,5 @@ export function logErrorBeforeThrow<T extends (...args: any[]) => any>(
 }
 
 export function coerceArray<T extends string>(param: T | T[] = []): T[] {
-  return [
-    ...new Set(toArray(param).flatMap((f: T) => f.split(',') as T[]) || []),
-  ];
+  return [...new Set(toArray(param).flatMap((f: T) => f.split(',') as T[]))];
 }
```

Comment:
There's no advantage to `coerceArray` accepting a type parameter - we don't call it ourselves, we pass it to Yargs' `coerce` option which is loosely typed. And the type assumptions are incorrect anyway - if you call `.split(',')` on `T`, the result won't be a `T[]`.

```ts
export function coerceArray(param: string | string[] = []): string[] {
  return [...new Set(toArray(param).flatMap(f => f.split(',')))];
}
```

Reply:
Good point